industrial_ci/ci_main.sh

```diff
@@ -38,6 +38,13 @@
 set -e # exit script on errors
 if [ "$DEBUG_BASH" ]; then set -x; fi # print trace if DEBUG
 
+#Define some verbose env vars
```

Comment:
Thanks for adding a comment. shell is too magic to me even now (and forever) so explanation per certain chunk of code is really helpful (I should've done that too...will enforce as my rule).